.github/workflows/integration_tests.yml

```diff
@@ -2,8 +2,6 @@ name: Integration tests
 on:
   pull_request:
     paths-ignore:
-    - '*.md'
-    - '**/*.md'
     - 'web/app/package.json'
```

Comment:
Should we remove this too?  If I understand correctly, paths-ignore and required github checks are completely incompatible.  Edits to `web/app/package.json` would exhibit this same problem.